### ISSUE TITLE
Can't run on Windows 10 with Python 2.7.13

### ISSUE THREAD

**Issue by gotounix** [wagtail/wagtail#3741]
Found a bug? Please fill out the sections below. 👍

### Issue Summary

Can't run on Windows 10 64 bit with Python 2.7.13 64 bit.

```
Traceback (most recent call last):
  File "manage.py", line 12, in <module>
    execute_from_command_line(sys.argv)
  File "F:\company\env\lib\site-packages\django\core\management\__init__.py", line 363, in execute_from_command_line
    utility.execute()
  File "F:\company\env\lib\site-packages\django\core\management\__init__.py", line 355, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "F:\company\env\lib\site-packages\django\core\management\base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "F:\company\env\lib\site-packages\django\core\management\commands\runserver.py", line 62, in execute
    super(Command, self).execute(*args, **options)
  File "F:\company\env\lib\site-packages\django\core\management\base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "F:\company\env\lib\site-packages\django\core\management\commands\runserver.py", line 101, in handle
    self.run(**options)
  File "F:\company\env\lib\site-packages\django\core\management\commands\runserver.py", line 110, in run
    autoreload.main(self.inner_run, None, options)
  File "F:\company\env\lib\site-packages\django\utils\autoreload.py", line 332, in main
    reloader(wrapped_main_func, args, kwargs)
  File "F:\company\env\lib\site-packages\django\utils\autoreload.py", line 303, in python_reloader
    exit_code = restart_with_reloader()
  File "F:\company\env\lib\site-packages\django\utils\autoreload.py", line 289, in restart_with_reloader
    exit_code = subprocess.call(args, env=new_environ)
  File "c:\python27\Lib\subprocess.py", line 168, in call
    return Popen(*popenargs, **kwargs).wait()
  File "c:\python27\Lib\subprocess.py", line 390, in __init__
    errread, errwrite)
  File "c:\python27\Lib\subprocess.py", line 640, in _execute_child
    startupinfo)
TypeError: environment can only contain strings
```

### Steps to Reproduce

An initial project, haven't edit any files.

### Technical details

python --version

```
Python 2.7.13
```

pip freeze

```
beautifulsoup4==4.6.0
certifi==2017.7.27.1
chardet==3.0.4
Django==1.11.3
django-modelcluster==3.1
django-taggit==0.22.1
django-treebeard==4.1.2
djangorestframework==3.6.3
html5lib==0.999999999
idna==2.5
olefile==0.44
Pillow==4.2.1
pytz==2017.2
requests==2.18.2
six==1.10.0
Unidecode==0.4.21
urllib3==1.22
wagtail==1.11.1
webencodings==0.5.1
Willow==0.4
```

**Comment by BertrandBordage** [comment]
Seems like an error related to Python & not directly Wagtail.

You probably have a non-ASCII character inside the path to your Wagtail project. Renaming the folder(s) containing unicode should fix the problame.

**Comment by gotounix** [comment]
@BertrandBordage The project is under F: root directory, doesn't have any non-ASCII character in the project.

**Comment by gasman** [comment]
I suspect this is https://code.djangoproject.com/ticket/28174 - upgrading to Python 3 or downgrading Django to 1.10.x should resolve it.

**Comment by BertrandBordage** [comment]
(Funny to have a Windows report by someone called @gotounix ^^)

**Comment by gotounix** [comment]
[#28174](https://code.djangoproject.com/ticket/28174) is fixed in the latest Django release 1.11.4, and everything will be OK.
